components/index.ts

```diff
@@ -1,96 +1,266 @@
-import * as Element from './element/index.js';
-import * as Field from './form/index.js';
-import * as Form from './form/index.js';
-import * as Format from './format/index.js';
+//--------------------------------------------------------------------//
+// Components
 
-export type { AlertProps } from './element/Alert';
-export type { BadgeProps } from './element/Badge';
-export type { LoaderProps } from './element/Loader';
-export type {
-  ModalContextProps,
-  ModalProviderProps, 
-  ModalProps
-} from './element/Modal';
+//accordion
 export type { 
+  AccordionStates,
+  AccordionConfig,
+  AccordionContextProps,
+  AccordionLabelProps,
+  AccordionContentProps,
+  AccordionBellowProps,
+  AccordionProps
+} from './Accordion.js';
+import Accordion, { 
+  useAccordion,
+  useAccordionContext,
+  AccordionContext,
+  AccordionActive,
+  AccordionInactive, 
+  AccordionLabel, 
+  AccordionContent, 
+  AccordionBellow
+} from './Accordion.js';
```

Comment:
No need to import sub components and functions.

`import Accordion from './Accordion.js';`